### PR TITLE
Code to add single organism to database at a time

### PR DESCRIPTION
**Pull request by WxnChen11**
Adds relevant CRISPR data to database for an organism (based on accession no.)

Pipeline:
1. Fetch sequence from ncbi
2. Save sequence as seq.fasta
3. Run seq.fasta through CRISPRCasFinder script
4. Save output of CRISPRCasFinder (in output_files/result.json) to database

### CODE REVIEW

File: add_organism.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    Organism,
+    Spacer,
+    Repeat,
+    LocusSpacerRepeat,
+    AntiCRISPR,
+    Locus
+)
+
+### Constants ###
+Entrez_email = 'wenxinchen11@hotmail.com'
```
Comment (mbonsma) [inline add_organism.py]: I would make this a command-line argument, you personally don't want to be responsible for ALL the downloads :)

File: add_organism.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+        # for testing only
+        pickle_item(res, accession)
+        return res
+
+    except Exception:
+        sys.stderr.write("Error! Cannot fetch: %s        \n" % accession)
+
+if __name__ == '__main__':
+    add_sequence_to_db(accession='NC_000853', name='Thermotoga maritima MSB8')
```
Comment (mbonsma) [inline add_organism.py]: Can you remove `name` as an argument and get the name from the fasta file directly? i.e. 

```
res = Entrez.efetch(db=db, rettype="gb", id=accession)

for record in parse(res, 'genbank'):
    name = record.annotations['organism'] 
```

Or you could skip adding the name entirely and just run [`organism_name_update.py`](https://github.com/phageParser/phageParser/blob/master/parserscripts/organism_name_update.py) afterwards.

Comment (mbonsma) [inline add_organism.py]: Also make the accession a command-line argument.

File: add_organism.py [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    print(o)
+    # Avoid Duplicate Entries (Comment out for testing)
+    if not created:
+        return created
+
+    # == for testing only ==
+    seq = get_pickle(accession)
+    if not seq:
+    # == for testing only ==
+        seq = get_ncbi_seq(Entrez_email, 'nuccore', 'fasta', accession)
```
Comment (mbonsma) [inline add_organism.py]: I'm getting an error: `name 'Entrez' is not defined` when I try this with a different accession (NC_015138)

File: add_organism.py (hunk 1 above)
Comment (WxnChen11) [reply add_organism.py]: Fixed :)

File: add_organism.py (hunk 2 above)
Comment (WxnChen11) [reply add_organism.py]: Fixed,  name is now grabbed from the fasta file

Comment (WxnChen11) [reply add_organism.py]: Fixed :)

File: add_organism.py (hunk 3 above)
Comment (WxnChen11) [reply add_organism.py]: Forgot to import Entrez, fixed

**Pull request merged**